integration/test_redis_specific.py

```diff
@@ -25,17 +27,24 @@ def redis():
     return get_redis_client()
 
 def test_redis_driver_write(redis):
+    random_name = ''.join(random.choice(string.ascii_lowercase) for i in range(5))
```

Comment:
Avoid using random without seed in tests as it makes tests nondeterministic.

Reply:
ok